src/DatePicker/index.js

```diff
@@ -0,0 +1,161 @@
+import delegate from 'delegate'
+import defaults from './lib/defaults'
+import zeroPad from './lib/zeroPad'
+
+export default class DatePicker {
+  constructor(el, options = {}) {
+    this.constants = Object.assign({}, defaults, options)
```

Comment:
Object.assign is still pretty new. What about pulling in https://github.com/babel/babel/tree/development/packages/babel-plugin-transform-object-assign ?